9.5/Dockerfile.rhel7

```diff
@@ -70,6 +72,12 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 
 VOLUME ["/var/lib/pgsql/data"]
 
+# {APP_DATA} needs to be accessed by postgres user while s2i assembling
+# postgres user changes permissiond of files in APP_DATA during assemblng
+RUN chown -R postgres:0 ${APP_DATA} && \
+    chmod -R ug+rwx ${APP_DATA} && \
+    usermod -a -G root postgres
```

Comment:
I still have the two questions:
- is the write permissions needed in case of postgresql?
- could we just drop chown/chmod and use `fix-permissions` above (where the data ownership is changed)?

Reply:
I think `fix-permissions` should be enough, at least it looks like it's enough for mysql: https://github.com/sclorg/mysql-container/blob/master/root-common/usr/libexec/container-setup#L57